src/Microsoft.AspNet.Mvc.TagHelpers/ImageTagHelper.cs

```diff
@@ -73,6 +79,9 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             {
                 // Pass through attribute that is also a well-known HTML attribute.
                 output.CopyHtmlAttribute(SrcAttributeName, context);
+
+                // Need to update the TagHelperOutput's attribute if the URL is application relative.
```

Comment:
`Content()` is called about here in the other two existing tag helpers. Why is it done above this block in this tag helper?